.ci/update-references.py

```diff
@@ -15,6 +16,10 @@
                 repo_dict["reference"] = REF
             else:
                 repo_dict.pop("reference", None)
+        if VERSION:
+            recipe["version"] = VERSION
+        else:
+            recipe.pop("version")
```

Comment:
I think this should be part of update, but maybe not? I dunno. More thought needed.